doc/middleware-it-registratore-telematico/installation/installation.md

```diff
@@ -0,0 +1,20 @@
+---
+slug: /poscreators/middleware-doc/italy/installation
+title: Installation
+---
+
+# Installation
+
+This chapter expands on the installation documentation covered in [Installation in General Part](../../general/installation/installation.md), with country-specific information applicable to the Italian market. 
+
+## fiskaltrust.Middleware
+
+### Windows, Linux, Mac
+
+The mentioned platforms use the fiskaltrust Launcher 2.0 to execute the fiskaltrust.SecurityMechanism. As the Version 2.0 of the Launcher is self-contained, it has native support for all mentioned platforms, which means there are no external dependencies in terms of third party software which needs to be installed.
+
+The following call parameters are available with the launcher `fiskaltrust.exe` for the Italian market (fiskaltrust.Middleware 1.3. and higher):
```

Comment:
The launcher 2.0 exe is `fiskaltrust.Launcher.exe` 😅 also the parameter below is a 1.3 parameter...

maybe we just link to the [launcher docs](https://github.com/fiskaltrust/middleware-launcher#getting-started) here?